node/subsystem/src/messages.rs

```diff
@@ -372,15 +374,21 @@ pub enum RuntimeApiRequest {
 	ValidatorGroups(RuntimeApiSender<(Vec<Vec<ValidatorIndex>>, GroupRotationInfo)>),
 	/// Get information on all availability cores.
 	AvailabilityCores(RuntimeApiSender<Vec<CoreState>>),
-	/// Get the global validation data.
-	GlobalValidationData(RuntimeApiSender<GlobalValidationData>),
-	/// Get the local validation data for a particular para, taking the given
+	/// Get the full validation data for a particular para, taking the given
```

Comment:
the "full validation data" is the next variant `FullValidationData` isn't it? This one is persisted?
```suggestion
	/// Get the persisted validation data for a particular para, taking the given
```